bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
@aantron Not public unfortunately - it's an internal tool.   I can make an example project under my github account though which illustrates the issue if that would help.